src/overlays/actors/ovl_En_Fish2/z_en_fish2.c

```diff
@@ -22,95 +65,1112 @@ const ActorInit En_Fish2_InitVars = {
+                this->unk_2C0 = 2;
+            }
+
+            if (gSaveContext.weekEventReg[81] & 0x40) {
+                this->unk_2C0 = 3;
+            }
+        } else {
+            if (gSaveContext.weekEventReg[81] & 0x80) {
+                this->unk_2C0 = 1;
+            }
+
+            if (gSaveContext.weekEventReg[82] & 1) {
+                this->unk_2C0 = 2;
+            }
+
+            if (gSaveContext.weekEventReg[82] & 2) {
+                this->unk_2C0 = 3;
+            }
+        }
+        cs = this->actor.cutscene;
+        this->unk_330 = D_80B2B370[this->unk_2C0];
+        i = 0;
+
+        // clang-format off
+        while (cs != -1) { this->unk_2BA[i] = cs; cs = ActorCutscene_GetAdditionalCutscene(cs); i++; }
+        // clang-format on
+
+        Collider_InitAndSetJntSph(globalCtx, &this->collider, &this->actor, &sJntSphInit, this->colliderElements);
+        this->collider.elements[0].dim.modelSphere.radius = 5;
+        this->collider.elements[0].dim.scale = 1.0f;
+        this->collider.elements[0].dim.modelSphere.center.x = 0x514;
+        this->collider.elements[0].dim.modelSphere.center.y = 0;
+        this->collider.elements[0].dim.modelSphere.center.z = 0;
+        this->collider.elements[1].dim.modelSphere.radius = 5;
+        this->collider.elements[1].dim.scale = 1.0f;
+        this->collider.elements[1].dim.modelSphere.center.x = 0x1F4;
+        this->collider.elements[1].dim.modelSphere.center.y = 0;
+        this->collider.elements[1].dim.modelSphere.center.z = 0;
+        this->actor.textId = 0x24C;
+        func_80B28B5C(this);
+    } else if (this->actor.params != 0) {
+        this->unk_2B4 = 10;
+        this->actor.draw = NULL;
+        this->actor.flags |= 0x8000000;
+        this->actionFunc = func_80B2A01C;
+    }
+}
+
+void EnFish2_Destroy(Actor* thisx, GlobalContext* globalCtx) {
+    EnFish2* this = THIS;
+
+    if (this->actor.params != 1) {
+        Collider_DestroyJntSph(globalCtx, &this->collider);
+    }
+}
+
+void func_80B287F4(EnFish2* this, s32 arg1) {
+    Vec3f sp2C;
+
+    if ((this->unk_354 != 0) && (this->unk_2C8 == 0)) {
+        if (this->unk_2C4 < 400) {
+            this->unk_2C4++;
+        }
+        this->unk_338 = 440.0f - this->unk_2C4;
+        if (arg1 == 0) {
+            this->unk_338 = 410.0f - this->unk_2C4;
+        }
+        Math_ApproachF(&this->unk_350->speedXZ, (D_80B2B380[0] - this->unk_330) * this->unk_338, 0.1f, 0.4f);
+    }
+    Math_Vec3f_Copy(&sp2C, &this->unk_350->world.pos);
+    this->unk_34A = Math_Vec3f_Yaw(&this->actor.world.pos, &sp2C);
+    this->unk_348 = Math_Vec3f_Pitch(&this->actor.world.pos, &sp2C);
+}
+
+s32 func_80B288E8(EnFish2* this, Vec3f vec, s32 arg2) {
+    f32 temp_f2 = this->unk_350->world.pos.x - vec.x;
+    f32 temp_f12 = this->unk_350->world.pos.y - vec.y;
+    f32 temp_f14 = this->unk_350->world.pos.z - vec.z;
+    f32 sqrt = sqrtf(SQ(temp_f2) + SQ(temp_f12) + SQ(temp_f14));
```

Comment:
```suggestion
    f32 dist = sqrtf(SQ(temp_f2) + SQ(temp_f12) + SQ(temp_f14));
```
I think `sqrt` is a bit too close to a keyword to be used as a variable